R/create_lkups.R

```diff
@@ -10,14 +10,14 @@ create_versioned_lkups <- function(data_dir) {
   data_dirs <- extract_data_dirs(data_dir = data_dir)
 
   versions <- names(data_dirs)
-  versions[1] <- "latest_release"
+  # versions[1] <- "latest_release"
 
   versions_paths <- purrr::map(data_dirs, create_lkups, versions = versions)
-
   names(versions_paths) <- versions
 
   return(list(versions = versions,
-              versions_paths = versions_paths))
+              versions_paths = versions_paths,
+              latest_release = versions[1]))
```

Comment:
We'll just need to be careful here once we start adding version folders for PPPs and testing. Related to the email Andres sent.

Reply:
Yep. Fully agree. We need to be very careful with the sorting and matching here. See also my commit here https://github.com/PIP-Technical-Team/pipapi/pull/219/commits/0f9192fa07f52bb2460d0670aa75fe2cb3a50d3d. We can modify this when we have agreed on the new folder structure.